terraform/modules/vpc/outputs.tf

```
@@ -9,5 +9,6 @@ output "outputs" {
     internet_egress_sg      = aws_security_group.internet_proxy_endpoint
     internet_proxy_endpoint = aws_vpc_endpoint.internet_proxy.dns_entry[0].dns_name
     s3_prefix_list_id       = module.vpc.s3_prefix_list_id
+    route_tables_private    = aws_route_table.private
```

or maybe it's this one that is misnamed and should actually be:
```suggestion
    route_table_private    = aws_route_table.private
```